1-Draft/RFC0002-Generalized-Splatting.md

```diff
@@ -160,7 +170,16 @@ and via splatting in the same invocation:
 # Must be an error, parse time or runtime, because startIndex
 # is specified positionally and via splatting.
 $subStringArgs = @{startIndex = 2}
-$str.SubString(2, @$subStringArgs)
+$str.SubString(3, @$subStringArgs)
+```
+
+Using the relaxed splatting operator, the `3` value will override the value in `$subStringArgs`:
+
+```PowerShell
+# This will not result in an error,
+# and the substring will be of length 3.
+$subStringArgs = @{startIndex = 2}
+$str.SubString(3, @?$subStringArgs)
```

Comment:
This would make the method resolution result unexpected -- it's too flexible, and probably not with much value. I don't think we should support relaxing splatting for method invocation.

Reply:
@PowerShell/powershell-committee agreed to defer relaxed splatting for method invocation.  Agreed to reuse existing PowerShell hashtable syntax rather than C# syntax or some new syntax for method invocation and named parameters:

foo.bar( @@{ index = 1; hello = "1" } )

Reply:
@SteveL-MSFT can we consider making the parentheses optional in that syntax? There are already way too many symbols going on there, and `.ForEach{}` is a thing already.

```
$Foo.Bar@@{ index = 1; hello = "1" }
```

Still not particularly liking the doubled up @ but c'est la vie I guess.